docs/pin.rst

```diff
@@ -224,4 +244,4 @@ in ``read_digital`` mode with the given pull mode.
     <https://microbit-micropython.readthedocs.io/en/latest/display.html#microbit.display.off>`_.
```

Comment:
A couple of lines above it mentions which pins are part of the display. Instead of having two lists here we should just mention the display pins are described in the table at the top.

Reply:
Done in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5